test/Engine/ProtoTest/TD/MultiLangTests/TestScope.cs

```diff
@@ -1042,7 +1042,7 @@ public void T032_Cross_Language_Variables()
 // expected: sum = 21
 // result: sum = 11";
             ExecutionMirror mirror = thisTest.RunScriptSource(src);
-            Assert.IsTrue((Int64)mirror.GetValue("sum").Payload == 11);
+            Assert.IsTrue((Int64)mirror.GetValue("sum").Payload == 21);
```

Comment:
Interesting, we expected 21, but the result was 11, so verified with 11 and made it pass.